doc/api/fs.md

```diff
@@ -3215,6 +3215,763 @@ changes:
+
+Asynchronous unlink(2). The `Promise` is resolved with no arguments upon
+success.
+
+### fs.promises.utimes(path, atime, mtime)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
+* Returns: {Promise}
+
+Change the file system timestamps of the object referenced by `path` then
+resolves the `Promise` with no arguments upon success.
+
+The `atime` and `mtime` arguments follow these rules:
+- Values can be either numbers representing Unix epoch time, `Date`s, or a
+  numeric string like `'123456789.0'`.
+- If the value can not be converted to a number, or is `NaN`, `Infinity` or
+  `-Infinity`, a `Error` will be thrown.
+
+### fs.promises.write(filehandle, buffer[, offset[, length[, position]]])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filehandle` {[FileHandle][]}
+* `buffer` {Buffer|Uint8Array}
+* `offset` {integer}
+* `length` {integer}
+* `position` {integer}
+* Returns: {Promise}
+
+Write `buffer` to the file specified by `filehandle`.
+
+The `Promise` is resolved with an object containing a `bytesWritten` property
+identifying the number of bytes written, and a `buffer` property containing
+the a reference to the `buffer` written.
+
+`offset` determines the part of the buffer to be written, and `length` is
+an integer specifying the number of bytes to write.
+
+`position` refers to the offset from the beginning of the file where this data
+should be written. If `typeof position !== 'number'`, the data will be written
+at the current position. See pwrite(2).
+
+The callback will be given three arguments `(err, bytesWritten, buffer)` where
+`bytesWritten` specifies how many _bytes_ were written from `buffer`.
+
+Note that it is unsafe to use `fs.write` multiple times on the same file
+without waiting for the `Promise` to resolve. For this scenario,
+`fs.createWriteStream` is strongly recommended.
+
+On Linux, positional writes don't work when the file is opened in append mode.
+The kernel ignores the position argument and always appends the data to
+the end of the file.
+
+### fs.promises.writeFile(file, data[, options])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `file` {string|Buffer|[FileHandle][]} filename or `FileHandle`
+* `data` {string|Buffer|Uint8Array}
+* `options` {Object|string}
+  * `encoding` {string|null} **Default:** `'utf8'`
+  * `mode` {integer} **Default:** `0o666`
+  * `flag` {string} **Default:** `'w'`
+
+Asynchronously writes data to a file, replacing the file if it already exists.
+`data` can be a string or a buffer.
+
+The `encoding` option is ignored if `data` is a buffer. It defaults
+to `'utf8'`.
+
+If `options` is a string, then it specifies the encoding. Example:
+
+Any specified `FileHandle` has to support writing.
+
+It is unsafe to use `fs.writeFile` multiple times on the same file without
+waiting for the `Promise` to be resolved (or rejected).
+
+If a `FileHandle` is specified as the `file`, it will not be closed
+automatically.
```

Comment:
"it will not be closed automatically" — if this is true, is it worth to be mentioned as an exception in the `FileHandle` description?

Reply:
good point, will need to figure out the different wording here. The `FileHandle` will be closed if it is allowed to gc .... but if a `FileHandle` is provided to this function, the function itself will not call `filehandle.close()`, it's still up to the user to do that.